src/index.js

```diff
@@ -32,12 +34,26 @@ const App = () => {
           actions.changeQuantity,
           actions.emptyCart,
           actions.readdProduct,
-          actions.readdOrder
+          actions.readdOrder,
+
+          actions.setCustomerId,
+          actions.removeCustomerId
```

Comment:
Remove these from the cart state listener. Its purpose is to recalculate the price when the cart changes...

Reply:
removed